docs/Quickstart.md

```diff
@@ -75,59 +72,7 @@ github "Anakros/WebRTC" == 63.11.20455
 github "socketio/Socket.IO-Client-Swift" == 9.0.1
 ```
 
-Check out the [integration example][13].
-
-## Setup
-
-To configure the SalemoveSDK, follow the steps:
-
-1. [Generating a universal push certificate](#certificate-generation)
-2. [Enabling the push capabilities](#push-notifications)
-3. [Configuring the project](#project-configuration)
-4. [Configuring the sdk](#sdk-configuration)
-
-### Push Notifications
-
-In order for the project to receive chat messages and other events, [push capabilities][3] should be enabled for the target. SaleMove will use your certificate when sending messages via the Apple Push Notification Service to the SaleMove client in your app.
-
-### Certificate generation
-
-Be sure to generate the **universal production** certificate.
-
-#### Fastlane
-
-The easies way to generate all the required files is to use [fastlane][1] built in tool called [pem][2].
-
-#### Submitting
-
-After the push certificate and the keys are generated, please submit *pem*, *p12* and *pkey* files to support@salemove.com.
-
-### Project Configuration
-
-The SalemoveSDK push service relies on [environment variables][4] so please make sure to add the `DEBUG` `1` to the run configuration. This will differentiate the gateway used to submit the messages.
-
-The SalemoveSDK uses the audio/video capabilities of the device. As such, the hosting app needs [NSCameraUsageDescription][11] and [NSMicrophoneUsageDescription][12] added to the plist file.
-
-The SalemoveSDK is not using Bitcode so this should be disabled for the project.
-
-### SDK Configuration
-
-he SalemoveSDK accepts three configuration options:
-
-- In order to configure the developer API token used by the SDK:
-```swift
-func configure(appToken: String)
-```
-
-- In order to configure the environment used by the SDK:
-```swift
-func configure(environment: String)
-```
-
-- In order to configure the site ID used by the SDK:
-```swift
-func configure(site: String)
-```
+Check out the [integration example][1].
```

Comment:
In general I recommend using named references instead of numerical as long as the named references do not distract from the text.